.claude/scripts/poll-pr.sh

```
@@ -1,191 +1,681 @@
+    2>/dev/null | jq -rs 'flatten | last | .body // ""' || echo '')
+
+  # Human PR comments (non-bot, non-our-own-account)
+  local human_comments_json self_login
+  self_login=$(gh api user --jq '.login' 2>/dev/null || echo '')
+  human_comments_json=$(gh api --paginate "repos/${REPO}/issues/${PR_NUMBER}/comments" \
+    --jq --arg self "$self_login" '[.[] | select(
+      (.user.login | endswith("[bot]") | not) and
+      .user.login != "openshift-ci-robot" and
+      ($self == "" or .user.login != $self) and
+      (.body | startswith("## Ready for Review") | not)
+    ) | {id: .id, login: .user.login, created_at: .created_at, body: .body}]' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)